lib/matplotlib/backends/backend_gtk3.py

```diff
@@ -821,6 +821,15 @@ def _normalize_shortcut(self, key):
 
         return ''.join(mods) + key
 
+    def _is_valid_shortcut(self, key):
+        if isinstance(key, str):
+            if 'cmd+' in key:
+                return False
+            if key.startswith('MouseButton.'):
+                return False
+            return True
```

Comment:
```suggestion
        if (isinstance(key, str) and
                (key.startswith('MouseButton.') or 'cmd+' in key)):
            return False
```
is IMHO more readable.

Reply:
Actually, this can be even simpler, since it doesn't need a `str`-check because, as @anntzer pointed out, the keymaps should not be `int`.